frontend/test/metabase/scenarios/dashboard/chained-filters.cy.spec.js

```diff
@@ -90,4 +190,54 @@ describe("scenarios > dashboard > chained filter", () => {
       });
     });
   }
+
+  context("reproduces metabase#13868", () => {
+    it.only("can use a chained filter with embedded SQL questions", () => {
+      createDashboardWithQuestion({}, dashboardId => {
+        cy.visit(`/dashboard/${dashboardId}`);
```

Comment:
Doesn't change the outcome of the test, but I was just wondering why do we visit the dashboard before the final `PUT` request?
I'd switch the order here, unless there's some very specific reason for it that I'm not seeing.

Reply:
Just a mistake, it's inconsequential to the functionality of the test, but I'll swap it (and add some comments) for correctness/clarity sake.